src/lib/modules/contracts_manager/index.js

```diff
@@ -220,6 +223,27 @@ class ContractsManager {
     );
   }
 
+  _emitContractLogError(address, contractName, functionAbi, inputs, errorMessage){
+    let idx = 0;
+    let paramString = '';
+    functionAbi.inputs.forEach((input) => {
+      let quote = input.type.indexOf("int") === -1 ? '"' : '';
+      paramString += quote + inputs[idx++] + quote + ", ";
+    });
+    paramString = paramString.substring(0, paramString.length - 2);
+    const log = {
+      type: 'contract-log',
+      address: address,
+      status: '0x0',
```

Comment:
I've seen us using `0x0` in other places as well. Maybe it's worth looking into making this a constant and reusing it in dedicated places?

Reply:
Have moved these to constants and updated references in the codebase where possible.